src/utilities/include/vec_dh.h

```diff
@@ -255,42 +232,19 @@ class ManagedVec {
     if (bytes >= (1ull << 63)) {
       throw std::bad_alloc();
     }
-#ifdef MANIFOLD_USE_CUDA
-    if (CudaEnabled())
-      cudaMallocManaged(reinterpret_cast<void **>(ptr), bytes);
```

Comment:
Oh, that's right - I forgot you already refactored VecDH for managed memory. Looks like you've taken care of everything I'd hoped for!